docs/README.md

```diff
@@ -0,0 +1,27 @@
+# Documentation
+
+## Scenarios
+
+The included AMIs built with this default repository consist of:
+
+- Broken Nginx
+- Broken Apache
+- Broken Python Web App
+- Broken Ruby Web App
+- Broken Bash Script
+
+### Broken Apache
```

Comment:
Missing summary for Broken Nginx too?

Reply:
I didn't want to commit this file, oops.